projects/exui-common-lib/src/lib/components/selected-case/selected-case.component.ts

```diff
@@ -56,10 +57,9 @@ export class SelectedCaseComponent implements OnInit {
         }
       }
     });
-    return canRemove;
+    return this.removeUserFromCaseToggleOn && canRemove;
```

Comment:
Not sure where the above code comes from, but this still has the racecondition in it, so this needs a refactor.

Reply:
Same goes for the methods below. You can't subscribe to an observable and run that code immediately to return something synchronously.